requirements.txt

```diff
@@ -2,7 +2,6 @@ argcomplete>=1.9.3
 boto3>=1.4.7
 durationpy>=0.5
 future>=0.16.0
```

Comment:
We should be able to remove `future` as well, no?

Reply:
Yes, changed that in #1998 though.

Reply:
Reverted on #1998 - both future and six will require more careful reviews for removal.

Reply:
I'd definitely like to clean up the imports to get rid of `future/past` cludge but I feel like that might be a bit further down the road. If we broke 2.7 already, just getting rid of it officially in the simplest manner makes the most sense to me.